services/uws-api-server/Chart.yaml

```diff
@@ -1,3 +1,7 @@
 apiVersion: v1
```

Comment:
Don't you also want to increase this to `v2` for Helm 3?

Reply:
Yes, you are correct.

Reply:
Fixed